test/circuit/library/test_raw_feature_vector.py

```diff
@@ -62,6 +62,10 @@ def test_fully_bound(self):
 
         self.assertEqual(bound.decompose(), ref)
 
+        # make sure that the bound circuit is a successful copy of the original circuit
+        self.assertEqual(circuit.num_qubits, bound.num_qubits)
+        self.assertEqual(circuit.feature_dimension, bound.feature_dimension)
```

Comment:
Isn't this covered implicitly by the test in L63? (At least the number of qubits)

Reply:
I'd prefer to keep these line to have explicit tests. L63 compares versus a reference circuit, while here is a direct comparison.